Make touchscreen application compatible with SDL1 and SDL2

Currently a touch screen is usable using 2 different kinds of events depending on SDL version:

- SDL1 (pygame 1.9.6): using event MOUSEBUTTONUP and MOUSEBUTTONDOWN
- SDL2 (pygame 2.0.1): using event FINGERUP and FINGERDOWN 

Managing the both kings of event permits to develop a software compatible with SDL1 and SDL2 library running on a touchscreen device.
However, `pygame-menu` does not permit it. I've got the following error when I set `touchscreen=True`.

```python
Traceback (most recent call last):
  File "/Volumes/DATA/Projects/pibooth/pibooth/booth.py", line 371, in main_loop
    self._menu = PiConfigMenu(self._pm, self._config, self, self._window)
  File "/Volumes/DATA/Projects/pibooth/pibooth/config/menu.py", line 101, in __init__
    onclose=self._on_close)
  File "/Users/antoine/.virtualenvs/pygame1/lib/python3.7/site-packages/pygame_menu/menu.py", line 568, in __init__
    assert version_major >= 2, 'touchscreen is only supported in pygame v2+'
AssertionError: touchscreen is only supported in pygame v2+
```

Is it possible to relax this constraints?

--- a/pygame_menu/menu.py
+++ b/pygame_menu/menu.py
@@ -563,15 +563,9 @@ class Menu(Base):
         self._mouse_visible_default = mouse_visible
 
         # Init touchscreen
-        if touchscreen:
-            version_major, _, _ = pygame.version.vernum
-            assert version_major >= 2, 'touchscreen is only supported in pygame v2+'
         if touchscreen_motion_selection:
             assert touchscreen, \
                 'touchscreen motion selection cannot be enabled if touchscreen is disabled'
-            assert hasattr(pygame, 'FINGERMOTION'), \
-                'pygame FINGERMOTION does not exist, thus, touchscreen motion ' \
-                'selection cannot be enabled'
         self._touchscreen = touchscreen
         self._touchscreen_motion_selection = touchscreen_motion_selection
 

--- a/test/_utils.py
+++ b/test/_utils.py
@@ -444,8 +444,6 @@ class PygameEventUtils(object):
         :param testmode: Event is in test mode
         :return: Event
         """
-        assert PYGAME_V2, \
-            'function only available in pygame v2+'
         if normalize:
             assert menu is not None, \
                 'menu reference must be provided if normalize is used (related to touch events)'

--- a/test/test_menu.py
+++ b/test/test_menu.py
@@ -1146,11 +1146,6 @@ class MenuTest(unittest.TestCase):
         """
         Test menu touchscreen behaviour.
         """
-        if not PYGAME_V2:
-            self.assertRaises(AssertionError,
-                              lambda: MenuUtils.generic_menu(title='mainmenu', touchscreen=True))
-            return
-
         self.assertRaises(AssertionError, lambda: MenuUtils.generic_menu(title='mainmenu', touchscreen=False,
                                                                          touchscreen_motion_selection=True))
         menu = MenuUtils.generic_menu(title='mainmenu', touchscreen=True, enabled=False)

